.github/workflows/plugin-server-ci.yml

```diff
@@ -1,12 +1,23 @@
-name: CI
+name: Plugin Server CI
 
 on:
-    - pull_request
+    pull_request:
+        paths:
+            - 'plugin-server/**'
+            - 'ee/models/**'
+            - 'ee/clickhouse/**'
+            - 'ee/migrations/**'
+            - 'posthog/models/**'
+            - 'posthog/migrations/**'
+            - 'posthog/plugins/**'
```

Comment:
plugin server tests are currently quite slow. speeding those up is something we should do, but nevertheless they don't need to _always_ run. here's the directory list I came up with of changes that would affect the plugin server

Reply:
Do we have to be concerned about Github required Checks? I put this in for backend but had to revert: https://github.com/PostHog/posthog/pull/6356

Instead using some paths-filter action to make the tests conditional.

Also, I guess changes to docker-compose might be relevant? Also, I think maybe it doesn't matter about posthog/models/** but just the migrations?

Reply:
Definitely not here if we'll probably not make these required for now at least.

Yeah fair point about migrations

Reply:
Addressed, thanks